test/Table_Tests/src/Common_Table_Operations/Order_By_Spec.enso

```diff
@@ -269,9 +269,7 @@ add_order_specs suite_builder setup =
 
         if setup.flagged ..Case_Insensitive_Ordering then group_builder.specify "should support case insensitive ordering" <|
             t1 = data.table.sort [..Name "eta"] text_ordering=(..Case_Insensitive)
-            expected = case setup.flagged ..Order_By_Unicode_Normalization_By_Default of
-                False -> ["Aleph", "alpha", "Beta", "bądź"]
-                True -> ["Aleph", "alpha", "bądź", "Beta"]
```

Comment:
What does snowflake do in this case?

Reply:
What has changed so that we can remove this difference?
I thought `Order_By_Unicode_Normalization_By_Default` was a difference between some backends, IIRC perhaps Postgres vs in-memory?

Reply:
This test isn't using default sorting. It specifies ..Case_Insensitive sorting. Order_By_Unicode_Normalization_By_Default refers to the default sort.